src/main/scala/org/apache/spark/streamdm/classifiers/trees/HoeffdingTree.scala

```diff
@@ -158,17 +164,17 @@ class HoeffdingTree extends Classifier {
 }
 
 class HoeffdingTreeModel(val espec: ExampleSpecification, val numericObserverType: Int = 0,
-  val splitCriterion: SplitCriterion = new InfoGainSplitCriterion(),
-  var growthAllowed: Boolean = true, val binaryOnly: Boolean = true,
-  val graceNum: Int = 200, val tieThreshold: Double = 0.05,
-  val splitConfedence: Double = 0.0000001, val learningNodeType: Int = 0,
-  val nbThreshold: Int = 0, val noPrePrune: Boolean = true,
-  val removePoorFeatures: Boolean = false, val splitAll: Boolean = false, val maxDepth: Int = 20)
-    extends ClassificationModel with Serializable with Logging {
+                         val splitCriterion: SplitCriterion = new InfoGainSplitCriterion(),
+                         var growthAllowed: Boolean = true, val binaryOnly: Boolean = true,
+                         val graceNum: Int = 200, val tieThreshold: Double = 0.05,
+                         val splitConfedence: Double = 0.0000001, val learningNodeType: Int = 0,
+                         val nbThreshold: Int = 0, val noPrePrune: Boolean = true,
+                         val removePoorFeatures: Boolean = false/*, val splitAll: Boolean = false*/, val maxDepth: Int = 20,
+                         val numSplitFeatures: Int = -1)
+  extends ClassificationModel with Serializable with Logging {
 
   type T = HoeffdingTreeModel
```

Comment:
T is not a good variable name.

Reply:
This needs to be named T, unless we modify streamdm.core.Learner. In practice, I could do that, but I would leave this kind of change to a separate PR.

Reply:
Ok 👍